.circleci/config.yml

```diff
@@ -83,6 +85,38 @@ jobs:
             # Slow, serial integration test, run nightly. Here to make sure the standard `mix test --only integration` works
             export SHELL=/bin/bash
             mix test --trace --only integration
+  barebuild_macos:
+    executor: metal_macos
+    environment:
+      MIX_ENV: test
+    steps:
+      - checkout
+      - run: echo 'export PATH=~/.cargo/bin:$PATH' >> $BASH_ENV
+      - run: |
+          brew install postgres
```

Comment:
wowow this is nice, could we do the same for the barebuild we do? maybe include this as part of `bin/setup` or is this because of `circle`?

Reply:
I saw these stuff (well, part of it) commented out in `bin/setup`, do you want me to resurrect it? (or resurrect it and guard it with env `NO_SETUP_BREW` or something)

Reply:
<3 yeah, I couldn't get it to work

Reply:
and then I just used postgres and elixir containers :)

Reply:
OK cool, I’ll update setup then.